package.json

```diff
@@ -111,6 +113,16 @@
 				"title": "Create Database"
 			},
 			{
+				"category": "DocumentDB",
+				"command": "cosmosDB.createDocDBDatabase",
+				"title": "Create Database"
+			},
+			{
+				"category": "DocumentDB",
+				"command": "cosmosDB.createDocDBCollection",
+				"title": "Create Collection"
```

Comment:
I only address DocumentDB atm. 
The others might need other API's for wrangling with the graphs and tables.

Reply:
The commands are shown only for DocumentDB account dbs/colls. 
I've made changes to that effect [here](https://github.com/Microsoft/vscode-cosmosdb/pull/93/files#diff-55f44dc2e4ecace970a4bac3d92a80ccR21)
I didn't do it for collection because we don't *yet* have any commands which ought to behave differently based on them